bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,48 @@
+from contextlib import suppress
+from typing import Optional
+
+from cowsay import char_names, get_output_string
+from discord import HTTPException
+from discord.ext import commands
+
+from bot.bot import Bot
+
+# Creates a local copy of char_names to filter out unsupported characters.
+LOCAL_CHAR_NAMES = list(char_names)
+for element in ["dragon", "trex", "stegosaurus", "turkey", "ghostbusters"]:
+    LOCAL_CHAR_NAMES.remove(element)
```

Comment:
Nitpick: Could this be changed to `SUPPORTED_CHAR_NAMES`, it sounds more appropriate then `LOCAL_CHAR_NAMES`